CHANGELOG.md

```
@@ -1,3 +1,658 @@
+  ([418e42f](https://github.com/gnosisguild/enclave/commit/418e42feceb5596a50f4f039f45d0f102fb7d50b))
+- upgrade to hardhat v3 and configure repo
+  ([#677](https://github.com/gnosisguild/enclave/issues/677))
+  ([7ccf6fa](https://github.com/gnosisguild/enclave/commit/7ccf6fa4d62a972a4d2336bd436d71bbc9b54535))
+- validate vote is <= balance ([#954](https://github.com/gnosisguild/enclave/issues/954))
+  ([bd5d35a](https://github.com/gnosisguild/enclave/commit/bd5d35ad7e8ec2fea7e8fd677ac24bf615fadd6b))
+- validate voting power ([#851](https://github.com/gnosisguild/enclave/issues/851))
+  ([d6e04bb](https://github.com/gnosisguild/enclave/commit/d6e04bb560f588c9f125dfef67bf9d27736011e2))
+- verify contracts on etherscan ([#867](https://github.com/gnosisguild/enclave/issues/867))
+  ([bd8d5bc](https://github.com/gnosisguild/enclave/commit/bd8d5bc95475859a42f5a55e37aa8581d4885df6))
+
+## [0.1.6](https://github.com/gnosisguild/enclave/compare/v0.1.2...v0.1.6) (2025-12-09)
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)